rust/processor/src/schema.rs

```diff
@@ -48,6 +48,8 @@ diesel::table! {
         transaction_timestamp -> Timestamp,
         inserted_at -> Timestamp,
         event_index -> Nullable<Int8>,
+        #[max_length = 66]
+        gas_fee_payer_address -> Nullable<Varchar>,
```

Comment:
May be this can be in a separate commit to keep things surgical? :)

Reply:
Yes but we need to deploy and backfill so 2 PRs add a lot more overhead. We can work around this but if it's just optics I'd like to move fast here.

Reply:
2 commits in the same PR?